pkg/virt-handler/node-labeller/node_labeller.go

```diff
@@ -433,3 +434,70 @@ func (n *NodeLabeller) loadKSM() {
 	n.KSM.Enabled = bytes.Equal(ksmValue, []byte("1\n"))
 	return
 }
+
+// shouldEnableKSM will update the ksm of the node (if available) based on the kv configuration.
+// If the node labels match the selector terms, the ksm will be enabled.
+// Empty Selector will enable ksm for every node
+func (n *NodeLabeller) shouldEnableKSM(node *v1.Node) {
+	ksmConfig := n.clusterConfig.GetKSMConfiguration()
+	if ksmConfig == nil {
```

Comment:
Is this needed? I believe `LabelSelectorAsSelector` will return a `selector` which will match nothing and therefore line 454 should take care of this.

Reply:
Without this line 448 will generate a nil pointer dereference error.
```
selector, err := metav1.LabelSelectorAsSelector(ksmConfig.NodeLabelSelector)
```